engine/src/main/scala/cromwell/engine/workflow/lifecycle/WorkflowFinalizationActor.scala

```diff
@@ -64,11 +64,11 @@ case class WorkflowFinalizationActor(workflowId: WorkflowId, workflowDescriptor:
   when(FinalizationPendingState) {
     case Event(StartFinalizationCommand, _) =>
       val backendFinalizationActors = Try {
-        (for {
-          backend <- workflowDescriptor.backendAssignments.values
-          props <- CromwellBackends.shadowBackendLifecycleFactory(backend).map(_.workflowFinalizationActorProps()).get
+        for {
+          (backend, calls) <- workflowDescriptor.backendAssignments.groupBy(_._2).mapValues(_.keys.toSeq)
+          props <- CromwellBackends.shadowBackendLifecycleFactory(backend).map(_.workflowFinalizationActorProps(workflowDescriptor.backendDescriptor, calls)).get
           actor = context.actorOf(props)
-        } yield (actor, backend)).toMap
+        } yield (actor, backend)
```

Comment:
ToL: I cannot find any use of the `backend`, and am exploring converting the `Map` to a `Set` in my ongoing final calls branch. I have two actors that run in the engine that are not backend actors, one for copying workflow outputs, one for copying workflow logs.